packages/twenty-ui/src/display/avatar/components/Avatar.tsx

```diff
@@ -101,14 +110,23 @@ export const Avatar = ({
   return (
     <StyledAvatar
       size={size}
-      backgroundColor={showBackgroundColor ? fixedBackgroundColor : 'none'}
+      backgroundColor={
+        Icon
+          ? theme.background.tertiary
+          : showBackgroundColor
+            ? fixedBackgroundColor
+            : 'none'
+      }
       color={fixedColor}
       clickable={!isUndefined(onClick)}
       rounded={type === 'rounded'}
+      type={type}
       onClick={onClick}
       backgroundTransparentLight={theme.background.transparent.light}
     >
-      {showPlaceholder ? (
+      {Icon ? (
+        <Icon color={iconColor ? iconColor : 'currentColor'} size={24} />
```

Comment:
We dont have 24 size in theme.icon.size so hardcoded here

Reply:
Use a multiple of theme.spacing, here it would be 6 I think.

Reply:
Ana is going to introduce 24px icons in https://github.com/twentyhq/twenty/pull/7093